src/main/webapp/WEB-INF/tags/instructor/course/commentArea.tag

```diff
@@ -0,0 +1,88 @@
+                                Students in this course
+                            </div>
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox answerCheckbox" type="checkbox" value="<%=CommentParticipantType.COURSE%>">
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox giverCheckbox" type="checkbox" value="<%=CommentParticipantType.COURSE%>">
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox recipientCheckbox" type="checkbox" value="<%=CommentParticipantType.COURSE%>" disabled="disabled">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="text-left">
+                            <div data-toggle="tooltip" data-placement="top" title="" data-original-title="Control what instructors can view">
+                                Instructors
+                            </div>
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox answerCheckbox" type="checkbox" value="<%=CommentParticipantType.INSTRUCTOR%>">
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox giverCheckbox" type="checkbox" value="<%=CommentParticipantType.INSTRUCTOR%>">
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox recipientCheckbox" type="checkbox" value="<%=CommentParticipantType.INSTRUCTOR%>">
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+        <textarea class="form-control" rows="6" placeholder="Enter your comment here ..." style="margin-bottom: 15px;"
+                name=<%=Const.ParamsNames.COMMENT_TEXT%> id="commentText"></textarea>
+        <div style="text-align: center;">
+            <input type="submit" class="btn btn-primary" id="button_save_comment" value="Save"> 
+            <input type="button" class="btn btn-default" id="button_cancel_comment" value="Cancel">
+            <input type="hidden" name=<%=Const.ParamsNames.COURSE_ID%> value="${courseId}">
+            <input type="hidden" name=<%=Const.ParamsNames.RECIPIENT_TYPE%> value="<%=CommentParticipantType.COURSE%>">
+            <input type="hidden" name=<%=Const.ParamsNames.RECIPIENTS%> value="${courseId}">
+            <input type="hidden" name=<%=Const.ParamsNames.COMMENTS_SHOWCOMMENTSTO%> value="">
+            <input type="hidden" name=<%=Const.ParamsNames.COMMENTS_SHOWGIVERTO%> value="">
+            <input type="hidden" name=<%=Const.ParamsNames.COMMENTS_SHOWRECIPIENTTO%> value="">
+            <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>" value="${googleId}">
```

Comment:
Ah, my previous comment meant that you could replace line 84 with:
`<input type="hidden" name="<%=Const.ParamsNames.USER_ID%>" value="${data.account.googleId}">`
since we can usually assume it exists.